### ISSUE TITLE
Twitter! - Big Brother 

### ISSUE THREAD

**Issue by BernardReyes** [Rucla-ed/rclass2#92]
Learned how accessible public data is. (!!!!!!!)

**Comment by ozanj** [comment]
yes, big brother is watching. and also, you could be big brother and use these skills and data to do research the field hasn't seen before. I think it is possible to do this with integrity.

**Comment by fernandacastellon** [comment]
SAME ! But are we able to pull data even if the twitter profile is private?

**Comment by ozanj** [comment]
Prior to reading this issue, I thought all tweets and all twitter accounts were public.

I did a quick skim of these three links:
- https://help.twitter.com/en/safety-and-security/public-and-protected-tweets
- https://help.twitter.com/en/safety-and-security/how-to-make-twitter-private-and-public
- https://www.businessinsider.com/how-to-make-twitter-private

my understanding [I'm guessing most of you already know this!] is that distinction is between "protected" and "unprotected" accounts.

if you have a protected account, your tweets can only be seen by your followers and you approve/deny any request by another twitter account to follow you.

I am unsure about the implications of this for getting data from Twitter API and for packages like rtweet. I am guessing that you can only grab data from unprotected twitter accounts or from protected accounts that you follow, but I haven't tested this out. If someone knows the actual answer, feel free to chime in
